app/domain/people/minimizer.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+#  Copyright (c) 2023, Pfadibewegung Schweiz. This file is part of
+#  hitobito and licensed under the Affero General Public License version 3
+#  or later. See the COPYING file at the top-level directory or at
+#  https://github.com/hitobito/hitobito.
+
+class People::Minimizer
+
+  def initialize(person)
+    @person = person
+  end
+
+  def run
+    # noop - override in wagon
```

Comment:
update minimized_at in here instead of further up?

Reply:
Yes! I realized this was necessary in the follow up ticket too so I'll change it here :)